app/src/main/java/com/hk/iwanttobesupermvp/model/GithubRepositoryModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.hk.iwanttobesupermvp.model
+
+import com.hk.iwanttobesupermvp.api.data.mock.MockDataDTO
+import com.hk.iwanttobesupermvp.contract.fragment.githubrepository.GithubRepositoryFragmentContract
+import javax.inject.Inject
+
+class GithubRepositoryModel @Inject constructor() :
+    GithubRepositoryFragmentContract.GithubRepositoryModel {
+
+    private var mockDataList = mutableListOf<MockDataDTO>()
+
+    override fun setMockData(_mockDataList: List<MockDataDTO>) {
+        mockDataList = _mockDataList as MutableList<MockDataDTO>
+    }
```

Comment:
```suggestion
    override fun setMockData(_mockDataList: List<MockDataDTO>) {
        mockDataList = _mockDataList.toMutableList()
    }
```

이런방식으로 하면 더 좋지 않을까요? 깊은 복사도 되고